engine/access/rpc/backend/backend_block_headers.go

```diff
@@ -23,7 +20,7 @@ func (b *backendBlockHeaders) GetLatestBlockHeader(ctx context.Context, isSealed
 		// get the latest seal header from storage
 		header, err = b.state.Sealed().Head()
 		if err != nil {
-			err = irrecoverable.NewExceptionf("failed to lookup sealed header: %w", err)
+			err = irrecoverable.NewExceptionf("failed to get sealed head: %w", err)
```

Comment:
let's use the original. avoid making unnecessary changes unless they meaningfully improve clarity or correctness